.github/workflows/shared_core_label.yml

```diff
@@ -0,0 +1,41 @@
+name: (Shared) Core Label
+
+on:
+  pull_request:
```

Comment:
This should be pull_request_target right? So you can't just add yourself to MAINTAINERS in a PR and get labeled as core

Reply:
Ah yeah good point, let me fix